smithy-diff/src/main/java/software/amazon/smithy/diff/evaluators/RemovedEntityBinding.java

```diff
@@ -37,6 +38,8 @@
 public final class RemovedEntityBinding extends AbstractDiffEvaluator {
     private static final String REMOVED_RESOURCE = "RemovedResourceBinding";
     private static final String REMOVED_OPERATION = "RemovedOperationBinding";
+    private static final String FROM_RESOURCE = ".FromResourceRemoved.";
```

Comment:
We already know it's removed from the name of the validator, so no need to repeat that in the secondary level. So I suggest ".Resource" and ".Service"

Reply:
I was worried that in the case of RemovedResourceBinding.Resource.MyResourceShape for example, it's a little misleading or unclear whether MyResourceShape is the parent or the child of the binding in question. What are your thoughts?

Reply:
Ah ok. Sure maybe then "FromResource"?

Reply:
Well actually the tricky part is that the shape I'm appending to the event id is the child, not the parent. I'm definitely open to suggestions for how to better word this though.